--- acct-6.3.5.orgi/configure.in	Sun Jul 21 19:39:18 2002
+++ acct-6.3.5/configure.in	Sun Jul 21 19:41:54 2002
@@ -91,51 +91,52 @@
 						AC_DEFINE(ACMEM_COMPT))
 		)
 		AC_HEADER_EGREP(comp_t,   sys/acct.h, AC_DEFINE(HAVE_COMP_T))
-		,
-  dnl
-  dnl here is the "else" clause for the above AC_CHECK_HEADER
-  dnl (for older versions of linux, which didn't have a sys/acct.h)
-  dnl
-  AC_CHECK_HEADER(linux/acct.h,
-		  AC_DEFINE(HAVE_LINUX_ACCT_H)
-		  DUMP_ACCT_PROG=dump-acct
-		  ACCTON_PROG=accton
-		  LASTCOMM_PROG=lastcomm
-		  SA_PROG=sa
-		  ACCTON_MAN=accton.8
-		  LASTCOMM_MAN=lastcomm.1
-		  SA_MAN=sa.8
-		  AC_HEADER_EGREP(ac_utime, linux/acct.h,
-				  AC_DEFINE(HAVE_ACUTIME)
-				  AC_HEADER_EGREP(comp_t.*ac_utime,
-						  linux/acct.h,
-						  AC_DEFINE(ACUTIME_COMPT))
-		  )
-		  AC_HEADER_EGREP(ac_stime, linux/acct.h,
-				  AC_DEFINE(HAVE_ACSTIME)
-				  AC_HEADER_EGREP(comp_t.*ac_stime,
-						  linux/acct.h,
-						  AC_DEFINE(ACSTIME_COMPT))
-		  )
-		  AC_HEADER_EGREP(ac_etime, linux/acct.h,
-				  AC_DEFINE(HAVE_ACETIME)
-				  AC_HEADER_EGREP(comp_t.*ac_etime,
-						  linux/acct.h,
-						  AC_DEFINE(ACETIME_COMPT))
-		  )
-		  AC_HEADER_EGREP(ac_io,    linux/acct.h,
-				  AC_DEFINE(HAVE_ACIO)
-				  AC_HEADER_EGREP(comp_t.*ac_io, linux/acct.h,
-						  AC_DEFINE(ACIO_COMPT))
-		  )
-		  AC_HEADER_EGREP(ac_mem,   linux/acct.h,
-				  AC_DEFINE(HAVE_ACMEM)
-				  AC_HEADER_EGREP(comp_t.*ac_mem, linux/acct.h,
-						  AC_DEFINE(ACMEM_COMPT))
-		  )
-		  AC_HEADER_EGREP(comp_t,   linux/acct.h, AC_DEFINE(HAVE_COMP_T))
-		  , )
 )
+#		,
+#  dnl
+#  dnl here is the "else" clause for the above AC_CHECK_HEADER
+#  dnl (for older versions of linux, which didn't have a sys/acct.h)
+#  dnl
+#  AC_CHECK_HEADER(linux/acct.h,
+#		  AC_DEFINE(HAVE_LINUX_ACCT_H)
+#		  DUMP_ACCT_PROG=dump-acct
+#		  ACCTON_PROG=accton
+#		  LASTCOMM_PROG=lastcomm
+#		  SA_PROG=sa
+#		  ACCTON_MAN=accton.8
+#		  LASTCOMM_MAN=lastcomm.1
+#		  SA_MAN=sa.8
+#		  AC_HEADER_EGREP(ac_utime, linux/acct.h,
+#				  AC_DEFINE(HAVE_ACUTIME)
+#				  AC_HEADER_EGREP(comp_t.*ac_utime,
+#						  linux/acct.h,
+#						  AC_DEFINE(ACUTIME_COMPT))
+#		  )
+#		  AC_HEADER_EGREP(ac_stime, linux/acct.h,
+#				  AC_DEFINE(HAVE_ACSTIME)
+#				  AC_HEADER_EGREP(comp_t.*ac_stime,
+#						  linux/acct.h,
+#						  AC_DEFINE(ACSTIME_COMPT))
+#		  )
+#		  AC_HEADER_EGREP(ac_etime, linux/acct.h,
+#				  AC_DEFINE(HAVE_ACETIME)
+#				  AC_HEADER_EGREP(comp_t.*ac_etime,
+#						  linux/acct.h,
+#						  AC_DEFINE(ACETIME_COMPT))
+#		  )
+#		  AC_HEADER_EGREP(ac_io,    linux/acct.h,
+#				  AC_DEFINE(HAVE_ACIO)
+#				  AC_HEADER_EGREP(comp_t.*ac_io, linux/acct.h,
+#						  AC_DEFINE(ACIO_COMPT))
+#		  )
+#		  AC_HEADER_EGREP(ac_mem,   linux/acct.h,
+#				  AC_DEFINE(HAVE_ACMEM)
+#				  AC_HEADER_EGREP(comp_t.*ac_mem, linux/acct.h,
+#						  AC_DEFINE(ACMEM_COMPT))
+#		  )
+#		  AC_HEADER_EGREP(comp_t,   linux/acct.h, AC_DEFINE(HAVE_COMP_T))
+#		  , )
+#)
 dnl
 dnl find out where utmp/pacct are stored
 dnl
